lib/Console/Traits/Parallelization.php

```diff
@@ -15,13 +15,17 @@
 
 namespace Pimcore\Console\Traits;
 
+use Closure;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputDefinition;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Lock\LockFactory;
 use Symfony\Component\Lock\LockInterface;
+use Webmozarts\Console\Parallelization\ErrorHandler\ErrorHandler;
+use Webmozarts\Console\Parallelization\ParallelExecutorFactory;
 
 trait Parallelization
```

Comment:
I think the `::configureParallelization()` might need another review, a few arguments & options changed or at least were added

Reply:
I created an issue https://github.com/pimcore/pimcore/issues/13724. thanks!